4_Zadania_dodatkowe/6.InfiniteSlider/js/app.js

```diff
@@ -0,0 +1,65 @@
+$(document).ready(function () {
+
+    var $prevButton = $('#prevPicture');
+    var $nextButton = $('#nextPicture');
+
+    var $liList = $('li');
+
+    var $ul = $('ul');
+
+    var visibleImageIndex = 0;
+    var imageWidth = $('img').width()+4;   // I tried with outerwidth, css(width) etc but still I have to add 4 because of the space between images; reset of all elements also didn't help
```

Comment:
see https://github.com/lukaszfranczak/jQueryTestRepo/pull/3#discussion_r188444341

Reply:
You're a genius :) I was trying to find out what's wrong but found only the outerWidth() which didn't work. Now it's finally working :)